.github/workflows/nestbuildmatrix.yml

```
@@ -8,7 +8,7 @@ on: [push, pull_request]
 
 jobs:
   clang-format:
-    runs-on: "ubuntu-20.04"
+    runs-on: "ubuntu-latest"
```

I think I'd like the explicit version number better:
```suggestion
    runs-on: "ubuntu-22.04"
```
Same below...